packages/mds-agency/utils.ts

```diff
@@ -429,33 +425,6 @@ export async function validateDeviceId(req: express.Request, res: express.Respon
   next()
 }
 
-export async function writeRegisterEvent(device: Device, recorded: number) {
-  const event: VehicleEvent = {
-    device_id: device.device_id,
-    provider_id: device.provider_id,
-    event_type: VEHICLE_EVENTS.register,
-    event_type_reason: null,
-    telemetry: null,
-    timestamp: recorded,
-    trip_id: null,
-    recorded,
-    telemetry_timestamp: undefined,
-    service_area_id: null
-  }
```

Comment:
Are `register` events now required rather than synthetic?

Reply:
Providers always used to `POST /vehicles` to register a vehicle and the handler would generate a corresponding `register` event. Are you saying this functionality is being removed?

Reply:
Yeah, it's being deprecated